src/sentry/static/sentry/app/components/dropdownButton.tsx

```diff
@@ -57,21 +69,21 @@ const StyledChevron = styled(IconChevron)`
 `;
 
 const StyledButton = styled(Button)<
-  Pick<Props, 'isOpen' | 'disabled' | 'hideBottomBorder'>
+  Required<Pick<Props, 'isOpen' | 'disabled' | 'hideBottomBorder' | 'priority'>>
 >`
   border-bottom-right-radius: ${p => (p.isOpen ? 0 : p.theme.borderRadius)};
   border-bottom-left-radius: ${p => (p.isOpen ? 0 : p.theme.borderRadius)};
   position: relative;
   z-index: 2;
   box-shadow: ${p => (p.isOpen || p.disabled ? 'none' : p.theme.dropShadowLight)};
-  border-bottom-color: ${p =>
-    p.isOpen && p.hideBottomBorder ? 'transparent' : p.theme.button.form.border};
```

Comment:
Do we no longer need to change the bottom border when the menu is open?

Reply:
we do, but I decided to replace it with   `&,` so we can reuse the `:active, :focus, :hover` logic